docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,356 @@
+in OpenShift, and that your tenant has access to them.
+
+Deploying compute nodes with SR-IOV networks is supported as a day 2, or post install, operation in the installer for both IPI and UPI workflows. Once you have verified that your OpenStack cluster can support SR-IOV in OpenShift, and have succesfully installed an OpenShift cluster that meets the [minimum requirements](README.md#openstack-requirements), you can follow these steps and example workflows to create compute nodes with SR-IOV NICs.
+
+## Preparing your OpenShift Cluster to use SR-IOV Networks
+
+Once your OpenShift control plane is running, you will need to install the SR-IOV Network Operator. To do so, you will need access to an account on your OpenShift cluster with `cluster-admin` privileges. Once you are logged into that account, follow [these steps](sriov-operator) to install the Operator.
+
+Next, you will need to create a machine config that enables the SR-IOV operator to get the UUID of networks from OpenStack. This machine config will allow the OpenStack metadata service to be available as a mountable drive.
+
+```yaml
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 20-mount-config
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+        - name: create-mountpoint-var-config.service
+          enabled: true
+          contents: |
+            [Unit]
+            Description=Create mountpoint /var/config
+            Before=kubelet.service
+
+            [Service]
+            ExecStart=/bin/mkdir -p /var/config
+
+            [Install]
+            WantedBy=var-config.mount
+
+        - name: var-config.mount
+          enabled: true
+          contents: |
+            [Unit]
+            Before=local-fs.target
+            [Mount]
+            Where=/var/config
+            What=/dev/disk/by-label/config-2
+            [Install]
+            WantedBy=local-fs.target
+```
+
+You can apply a machine config to your machine pool that enables the No-IOMMU feature for the Red Hat OpenStack Platform (RHOSP) virtual function I/O (VFIO) driver. The RHOSP vfio-pci driver requires this feature.
+
+```yaml
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 99-vfio-noiommu
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - filesystem: root
+        path: "/etc/modprobe.d/vfio-noiommu.conf"
+        contents:
+          source: data:text/plain;charset=utf-8;base64,b3B0aW9ucyB2ZmlvIGVuYWJsZV91bnNhZmVfbm9pb21tdV9tb2RlPTEK
+          verification: {}
+        mode: 0644
+```
```

Comment:
do we need to also need to Enabling the RHOSP metadata service as a mountable drive ? 
(https://deploy-preview-28315--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-user-sr-iov.html#networking-osp-enabling-metadata_installing-openstack-user-sr-iov)

Reply:
yes, I believe we do.